src/main.ts

```diff
@@ -94,12 +94,12 @@ async function run(): Promise<void> {
       mkdir -p ${sarifOutput};
     fi
     sed -i "s,project:.*,project: ${repo.toLowerCase()},g" ${mayhemfile};
-    image_found=$(grep "image: " ${mayhemfile});
-    if [ -z "$image_found" ]; then
+    image_line=$(grep "image: " ${mayhemfile});
+    if [ -n "$image_line" ]; then
+      sed -i -E "s/#\s+image:.+|image:.+/image: ${image}/g;" ${mayhemfile};
+    else
```

Comment:
flipped the conditional to make this more readable (i.e., if an `image` parameter is found, modify the `image` value, otherwise add the `image` field entirely to the Mayhemfile)